lib/triangle.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {isTriangle}
+
+function isTriangle(x, y, z) {
+  return ((x + y > z) && (x + z > y) && (y + z > x))
```

Comment:
What would happen if we had a negative side or a side equal to 0? These conditions could still return true